template/package.json

```diff
@@ -53,7 +53,7 @@
       "hover": [
         {
           "action": "findImplementations",
-          "when": "get(context, `implementations_${resource.language}`) && (goToDefinition.showLoading || goToDefinition.url || goToDefinition.error)"
+          "when": "resource.language == 'LANGID' && get(context, `implementations_${resource.language}`) && (goToDefinition.showLoading || goToDefinition.url || goToDefinition.error)"
```

Comment:
I wonder if we also want to include this condition with the `menus.panel/toolbar` item above? Basically anything that defines a when condition here will be exported as _any_ language extension so there might be other collisions.

Reply:
I think so! Then we could get rid of the band-aid action deduplication in the `<Panel>` component